work.py

```diff
@@ -374,29 +384,33 @@ def sync_timesheetline(cls, activities):
         TimesheetLine = pool.get('timesheet.line')
         Warning = pool.get('res.user.warning')
 
+        if Transaction().context.get('skip_sync_timesheetline', False):
```

Comment:
At the time of making a copy, we enter a loop because the id (activity or project) is different